scriptmodules/system.sh

```diff
@@ -349,3 +352,11 @@ function platform_imx6() {
     __default_makeflags="-j2"
     __platform_flags="arm armv7 neon"
 }
+
+function platform_vero4k() {
+    __default_cflags="-I/opt/vero3/include -L/opt/vero3/lib -O3 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations"
```

Comment:
Would prefer -O2 for default optimisation level (many of the emulators override it anyway)

Reply:
Done.